int/api/html/front/website.js

```diff
@@ -162,7 +162,9 @@ async function tableInsert(resp, count) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = `//${resp.name}.massa:${window.location.port}/`;
+    const url = isWindowsAndFirefox()
+        ? `http://${resp.name}.massa:${window.location.port}/`
+        : `https://${resp.name}.massa:${window.location.port}/`;
```

Comment:
Duplicated code here, maybe you could add a function `getUrl()`

Reply:
same as above